wp/resources/sample_binaries/nested_ifs/run_wp_inline.sh

```diff
@@ -8,7 +8,7 @@
 run () {
   bap wp \
     --func=main \
-    --inline=.* \
+    --inline=calloc \
```

Comment:
Is this a fix for an older issue?

Reply:
Since I removed VERIFIER_nondet from the default list of function specs, inlining everything, would inline calloc, and that would make this test fail. When I add the nondet spec back to the default, I think I could change this back to `.*`.

Reply:
👍